### ISSUE TITLE
Qbittorent or Filebrowser Issue ‼️

### ISSUE THREAD

**Issue by prashantdwivedi** [getumbrel/umbrel#1765]
Hi, team. You guys are doing an incredible job of creating such an outstanding OS.

**I get problems after a couple of Install a fresh OS.**

```
1: Attempt
After Fresh Install OS
1- Install Filebrowser
2- Install qbitttorrent

Add torrent and let to download but cannot access that downloaded file from filebrowser.
```

```
2: Attempt
After Fresh Install OS
1- Install qbitttorrent
2- Install Filebrowser

Add torrent and let to download but cannot access that downloaded file from filebrowser.

```
Please FIX it. ISSUE

**Suggestion:** Please make a **DEFAULT** umbrella File browser like **CASA OS** has.

Current filebrowser is Pain in Your OS.
The filebrowser is broken and not able to access through the root.

Regard
Umbrel OS ❤️

**Comment by prashantdwivedi** [comment]
I haven't received any response from the developer yet. Can you please help me with this?

**Comment by highghlow** [comment]
QBittorrent probably downloads all media into it's app directory (`~/umbrel/app-data/<app_id>`), while filebrowser provides access to the umbrel data dir (`~/umbrel/data/storage`). I'll inspect qbittorrent's docker-compose and fix it. Right now you could use transmission as you bittorrent client, or use ssh to copy the files

**Comment by highghlow** [comment]
QBittorrent mounts $UMBREL_ROOT/downloads to /downloads. All files should be downloaded to directory

**Comment by highghlow** [comment]
> You need to make it a file browser as well.

It already exists as an app

**Comment by prashantdwivedi** [comment]
> > You need to make it a file browser as well.
> 
> It already exists as an app

No, dude. You have to install the `File Browser` app, and that file browser does not get root access.  
Please try 'Casa OS`. It is better than the `file default` app and comes with default.

If I have to do manual stuff, why use `umbrel`? Then, you must remove the line `no technical knowledge required` from the website.
![CleanShot 2024-04-13 at 19 18 57@2x](https://github.com/getumbrel/umbrel/assets/6306052/e0fb2a34-a9ec-4390-ae94-15803e008a02)

Try `CasaOS`. It is called "no technical knowledge required"; now my love for umbrel is 💔 break.

**Comment by highghlow** [comment]
Well, Umbrel is still developing. Linux support was added less than a month ago.

(It also requires much less technical knowledge than running everything yourself)

Also, you should probably mark the issue as closed

**Comment by highghlow** [comment]
Hey, @lukechilds, can you close this issue? (works as intended)

**Comment by nmfretz** [comment]
Thanks for reporting this @prashantdwivedi. I have updated qBittorrent with a fix: https://github.com/getumbrel/umbrel-apps/pull/893#issuecomment-2149633385

 > Suggestion: Please make a DEFAULT umbrella File browser like CASA OS has.
 
 Thanks for the suggestion. Stay tuned 😉
